internal/database/conf.go

```diff
@@ -117,6 +135,64 @@ func (s *confStore) SiteGetLatest(ctx context.Context) (_ *SiteConfig, err error
 	return tx.getLatest(ctx)
 }
 
+func (s *confStore) ListSiteConfig(ctx context.Context, opt SiteConfigListOptions) ([]*SiteConfig, error) {
+	// Ascending order by default.
+	orderBy := AscendingOrderByDirection
+	if opt.OrderByDirection != "" {
+		orderBy = opt.OrderByDirection
+	}
+
+	q := sqlf.Sprintf(`
+		SELECT
+			 id,
+			 author_user_id,
+			 type,
+			 contents,
+			 created_at,
+			 updated_at
+		 FROM critical_and_site_config
+         WHERE type = 'site'
+		 ORDER BY id `+string(orderBy)+`
```

Comment:
```suggestion
		 ORDER BY id %s
```

and put the `orderBy` into the args to `Sprintf` to be consistent and not do string concatenation with SQL.